integrations/dd-sdk-android-compose/src/main/kotlin/com/datadog/android/compose/InteractionTracking.kt

```diff
@@ -244,35 +246,38 @@ internal suspend fun <T> trackDragInteraction(
 ) {
     val ongoingInteractions = mutableMapOf<DragInteraction.Start, T>()
     try {
-        interactionSource.interactions.collect { interaction ->
-            when (interaction) {
-                is DragInteraction.Start -> {
-                    @Suppress("UnsafeThirdPartyFunctionCall")
-                    onStart(ongoingInteractions, interaction)
-                }
-                is DragInteraction.Stop, is DragInteraction.Cancel -> {
-                    val start = when (interaction) {
-                        is DragInteraction.Stop -> interaction.start
-                        is DragInteraction.Cancel -> interaction.start
-                        else -> {
-                            // this will never happen, let's just make compiler happy and
-                            // give a fake start
-                            Log.e(
-                                LOG_TAG,
-                                "Unexpected branch reached" +
-                                    " for the drag interaction start"
-                            )
-                            DragInteraction.Start()
-                        }
+        interactionSource.interactions.collect(
+            FlowCollector { interaction ->
```

Comment:
strange that it compiled. probably it is pure recent Android Studio thing (for sure it didn't complain before), which is not capable to understanding that `FlowCollector` is functional interface (but with `suspend` method), while Kotlin compiler is ok with that.